api/core/embedding/cached_embedding.py

```diff
@@ -110,6 +110,7 @@ def embed_query(self, text: str) -> list[float]:
             embedding_results = embedding_result.embeddings[0]
             embedding_results = (embedding_results / np.linalg.norm(embedding_results)).tolist()
         except Exception as ex:
+            logging.exception(f"Failed to embed query text: {ex}")
```

Comment:
For new exceptions, please add a `DEBUG` check. Eg. if DEBUG then log something.

Reply:
@crazywoola done, pls help me to review again.